packages/x-charts/src/internals/components/ChartsText.tsx

```diff
@@ -93,7 +93,12 @@ export function ChartsText(props: ChartsTextProps) {
       style={style}
     >
       {wordsByLines.map((line, index) => (
-        <tspan x={x} dy={`${index === 0 ? startDy : wordsByLines[0].height}px`} key={index}>
+        <tspan
+          x={x}
+          dy={`${index === 0 ? startDy : wordsByLines[0].height}px`}
+          dominantBaseline={dominantBaseline}
```

Comment:
```suggestion
          dominantBaseline={dominantBaseline} // Propagated to fix Safari issue: https://github.com/mui/mui-x/issues/10808
```

I thought about adding it in the code such that in a year can still see why it's here